src/components/ThemeProvider/utils/__tests__/themeDefaultStyle.test.ts

```diff
@@ -0,0 +1,18 @@
+import themeDefaultStyle from '../themeDefaultStyle'
```

Comment:
I think the license header is required for tests as well, not only impl files

Reply:
:+1: